src/lib/plot/series/line-series.js

```diff
@@ -36,15 +36,27 @@ const STROKE_STYLES = {
 
 const defaultProps = {
   strokeStyle: 'solid',
-  opacity: 1
+  opacity: 1,
+  curve: null
 };
 
 const propTypes = {
   ...AbstractSeries.propTypes,
-  strokeStyle: React.PropTypes.oneOf(Object.keys(STROKE_STYLES))
+  strokeStyle: React.PropTypes.oneOf(Object.keys(STROKE_STYLES)),
+  curve: React.PropTypes.string
 };
 
 class LineSeries extends AbstractSeries {
+
+  _renderLine(data, x, y, curve) {
+    let line = d3Shape.line();
+    if (curve !== null && d3Shape[curve]) {
+      line = line.curve(d3Shape[curve]);
```

Comment:
Was looking through the curve documentation. Any thoughts on how to specify `var line = d3.line().curve(d3.curveCatmullRom.alpha(0.5));` with the curve prop?